tests/ruby_spec.lua

```diff
@@ -0,0 +1,50 @@
+describe('linter.ruby', function()
+  it('can parse the output', function()
+    local parser = require('lint.linters.ruby').parser
+    local bufnr = vim.uri_to_bufnr('file:///foo.rb')
+    local output = [[
+/foo.rb:2: warning: key :bar is duplicated and overwritten on line 2
+/foo.rb:2: warning: unused literal ignored
+/foo.rb:3: syntax error, unexpected end-of-input
```

Comment:
The output is the result of running ruby linter on the following code.

```ruby
# sample code containing warning and syntax error
{ bar: 'baz1', bar: 'baz2' }
def
```

Execution result:

```sh
$ ruby -w -c foo.rb
foo.rb:2: warning: key :bar is duplicated and overwritten on line 2
foo.rb:2: warning: unused literal ignored
foo.rb:3: syntax error, unexpected end-of-input
```